Pages/Index.cshtml

```diff
@@ -1,7 +1,7 @@
 @page
 @model IndexModel
 @{
-    ViewData["Title"] = "Home Page";
+    ViewData["Title"] = "Property Home Page";
```

Comment:
Specifying Property Homepage Name to be more accessible in Index.cshtml

Reply:
Good suggestion, and I'd go a step further.  I'd take "Home Page" out entirely.  Just put the whole project name in here.  "Home Page" is a bit 1990s, and feels redundant.  Imagine if every browser tab in your browser had the word "Home Page" affixed to it.